### ISSUE TITLE
Signup - terms of use

### ISSUE THREAD

**Issue by pscomgr** [iNextrix/ASTPP#539]
Hi,

is it possible to make a choice for users to accept the terms of use upon signup?
it can be a link or a window that has all the terms in.

Kind regards

**Comment by devangn** [comment]
@pscomgr Yes its possible but we haven't got much requests like this so we haven't consider it yet, but yes.
